README.md

```diff
@@ -1 +1,158 @@
-# Final-capstone-Frontend
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] In this review iteration, one test is failing:

![image](https://github.com/tajulafreen/Final-capstone-Frontend/assets/57421823/8c87cd10-f06e-413b-8558-095065dc218d)

Kindly correct it. 👍